gateway/src/node/client_handling/websocket/connection_handler/authenticated.rs

```
@@ -334,12 +351,18 @@ where
             let aggregated_verification_key = self
                 .inner
                 .shared_state
-                .coconut_verifier
+                .ecash_verifier
                 .verification_key(credential.data.epoch_id)
                 .await?;
 
-            let params = bandwidth_credential_params();
-            if !credential.data.verify(params, &aggregated_verification_key) {
+            //4 is legacy from old coconut credential
```

should we even be handling those requests in the first place? maybe just return an error that they're no longer supported

---

Goes with the whole backward compatibility discussion

---

TO FIX : scrape backward compatibility

---

Removed in c6c3b2f644b6c608f9f90ea3bd75aded0d45d01e